easybuild/easyconfigs/a/ant/ant-1.10.0-Java-1.8.0_112.eb

```diff
@@ -0,0 +1,25 @@
+name = 'ant'
+version = '1.10.0'
+versionsuffix = '-Java-%(javaver)s' 
+
+homepage = 'http://ant.apache.org/'
+description = """Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files
+ as targets and extension points dependent upon each other. The main known usage of Ant is the build of Java applications."""
+
+toolchain = {'name': 'dummy', 'version': ''}
```

Comment:
I thought we had to put `dummy` in the version field too. 

Has that changed now?